src/probnum/filtsmooth/statespace/sde_utils.py

```diff
@@ -0,0 +1,26 @@
+"""Continuous-time transition utility functions."""
+
+
+import numpy as np
+import scipy.linalg
+
+
+def matrix_fraction_decomposition(driftmat, dispmat, dt):
```

Comment:
Do you think a check for the value of `dt` would be reasonable?

Reply:
What sort of check do you mean? That it is a float? I dont know, I tend to avoid checks unless there is some really weird behaviour that needs to be caught. 
I guess in line 21 below, if `dt` was not a float, there would be a reasonable error. what do you think? 


Also, I do not expect that MFD is used heavily by an outside user...

Reply:
> What sort of check do you mean?
Not really a type check but rather, if someone passes `dt = t_i - t_j` and it accidentally results in a negative `dt`, that might result in weird behavior that is difficult to debug.

However, ... 
> Also, I do not expect that MFD is used heavily by an outside user...
... that's probably true.